build.sbt

```diff
@@ -423,6 +424,7 @@ lazy val sdk = project
   .settings(
     coverageFailOnMinimum := false,
     libraryDependencies  ++= Seq(
+      akkaActor,
```

Comment:
why this dependency?

Reply:
Cause when building an akka `Uri` we need akka actor, otherwise it blows up

Reply:
I've added it to the test scope, but it so gonna crash anyway using it from the outside if akkaActor is not present.